docs/stacks-guide-gcp.md

```diff
@@ -0,0 +1,424 @@
+```
+
+## Configure Provider Resources
+
+Once we have the network set up, we also need to tell Crossplane how to
+satisfy WordPress's claims for a database and a Kubernetes cluster.
+The resource classes serve as template for the new claimswe make. 
+The following resource classes allow the GKECluster and CloudSQL claims
+to be satisfied with the network configuration we just set up:
+
+```
+cat > environment.yaml <<EOF
+---
+apiVersion: database.gcp.crossplane.io/v1alpha2
+kind: CloudsqlInstanceClass
+metadata:
+  name: standard-cloudsql
+  namespace: gcp
+specTemplate:
+  databaseVersion: MYSQL_5_7
+  tier: db-n1-standard-1
+  region: us-central1
+  storageType: PD_SSD
+  storageGB: 10
+  # Note from GCP Docs: Your Cloud SQL instances are not created in your VPC network.
+  # They are created in the service producer network (a VPC network internal to Google) that is then connected (peered) to your VPC network.
+  privateNetwork: projects/$PROJECT_ID/global/networks/example-network
+  providerRef:
+    name: gcp-provider
+    namespace: gcp
+  reclaimPolicy: Delete
+---
+apiVersion: compute.gcp.crossplane.io/v1alpha2
+kind: GKEClusterClass
+metadata:
+  name: standard-gke
+  namespace: gcp
+specTemplate:
+  machineType: n1-standard-1
+  numNodes: 1
+  zone: us-central1-b
+  network: projects/$PROJECT_ID/global/networks/example-network
+  subnetwork: projects/$PROJECT_ID/regions/us-central1/subnetworks/example-subnetwork
+  enableIPAlias: true
+  clusterSecondaryRangeName: pods
+  servicesSecondaryRangeName: services
+  providerRef:
+    name: gcp-provider
+    namespace: gcp
+  reclaimPolicy: Delete
+EOF
+
+kubectl apply -f environment.yaml
+# Example YAML exists in https://github.com/crossplaneio/crossplane/blob/master/cluster/examples/workloads/kubernetes/wordpress/gcp/environment.yaml as well.
+```
+
+The steps that we have taken so far have been related to things that can
+be shared by all resources in all namespaces of that Crossplane cluster.
+Now, we will keep going with creating an app namespace and populating it
+with resources that will help Crossplane know with what configuration it
+should satisfy the claims. You can use any namespace for your app's
+resources but for this tutorial we'll create a new namespace.
+
+```
+kubectl create namespace mynamespace
+```
+
+Now we need to tell Crossplane which resource classes should be used to
+satisfy our claims in that app namespace. We will create portable
+classes that have have reference to non-portable ones that we created
+earlier. In our claims, we can refer to those portable classes directly
+or label one as the default portable class to be used in claims that do
+not have class reference.
+
+> Portable classes are a way of referring to non-portable resource classes in other namespaces. 
```

Comment:
May need to update if we use terminology other than "non-portable" in user-facing docs.

Reply:
Good point. Let's keep track of this and consider it for a future changeset